frappe/tests/test_commands.py

```diff
@@ -533,8 +533,8 @@ def test_backup_fails_with_exit_code(self):
 			frappe.conf.db_name,
 			frappe.conf.db_name,
 			frappe.conf.db_password + "INCORRECT PASSWORD",
-			db_host=frappe.db.host,
-			db_port=frappe.db.port,
+			db_host=frappe.conf.db_host,
+			db_port=frappe.conf.db_port,
```

Comment:
these and similar are now taking the defaults from `libpg` and [`pymysql` here](https://github.com/PyMySQL/PyMySQL/blob/f4c348fdcf4ac21a92be58b6f94e9d7a13826a38/pymysql/connections.py#LL291C1-L292C33)